index.js

```diff
@@ -0,0 +1,110 @@
+const express = require('express');
+const helmet = require('helmet');
+
+const db = require('./data/db');  
+
+const server = express();
+
+server.listen(8000, () => console.log('API running...'));
+
+server.use(helmet());
+server.use(express.json());
+
+// post - route handlers
+server.post('/api/users', (req, res) => {
```

Comment:
Nice work with post. Just need to add validation handling if no name or bio is provided.